### PR TITLE
Switch from Alpine to Debian as the default base image.

### PR DESCRIPTION
**Pull request by xnull**
Ubuntu is a more convenient image for us to use.
Also ubuntu-based image is using in the private corfu version.

Also, the previous image was missing several native libraries, including ld-linux-x86-64.so.

## Overview

Description:

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

**Review by PavelZaytsev — APPROVED** [review]

File: .ci/infrastructure-docker-build.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set -e
 
 PROFILE=$1
+BASE_IMAGE=$2
```
Comment (chetangudisagar) [inline .ci/infrastructure-docker-build.sh:6]: This line and its associated `if` block is never being called/used in the current repo as the second argument is never passed. Is this being used in any other repos? If not, why is this required?

**Review by chetangudisagar — APPROVED** [review]
LGTM except for the query on `.ci/infrastructure-docker-build.sh` changes.
